<ui version="4.0">
 <class>DlgAddImageClass</class>
 <widget class="QDialog" name="DlgAddImageClass">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>280</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy vsizetype="Fixed" hsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>280</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>400</width>
    <height>280</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Add Images...</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QGridLayout">
   <item row="0" column="0">
    <layout class="QVBoxLayout">
     <item>
      <layout class="QHBoxLayout">
       <item>
        <widget class="QGroupBox" name="grpSelImage">
         <property name="title">
          <string>Select Images</string>
         </property>
         <layout class="QGridLayout">
          <item row="0" column="0">
           <layout class="QHBoxLayout">
            <item>
             <layout class="QVBoxLayout">
              <property name="spacing">
               <number>2</number>
              </property>
              <item>
               <widget class="QListWidget" name="lstImages" />
              </item>
              <item>
               <widget class="QLabel" name="lbCount">
                <property name="text">
                 <string>%1 images selected</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignCenter</set>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QVBoxLayout">
              <item>
               <widget class="QPushButton" name="btAddImages">
                <property name="sizePolicy">
                 <sizepolicy vsizetype="Fixed" hsizetype="Fixed">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="text">
                 <string>Add Images</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QCheckBox" name="ckSaveRel">
                <property name="text">
                 <string>Use relative image path</string>
                </property>
               </widget>
              </item>
              <item>
               <spacer>
                <property name="orientation">
                 <enum>Qt::Vertical</enum>
                </property>
                <property name="sizeHint">
                 <size>
                  <width>20</width>
                  <height>40</height>
                 </size>
                </property>
               </spacer>
              </item>
             </layout>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <layout class="QVBoxLayout">
         <item>
          <widget class="QDialogButtonBox" name="buttonBox">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="standardButtons">
            <set>QDialogButtonBox::Cancel|QDialogButtonBox::NoButton|QDialogButtonBox::Ok</set>
           </property>
          </widget>
         </item>
         <item>
          <spacer>
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QGroupBox" name="grpFilePath">
       <property name="title">
        <string>Annotation File Path</string>
       </property>
       <layout class="QGridLayout">
        <item row="0" column="0">
         <layout class="QHBoxLayout">
          <item>
           <widget class="QComboBox" name="cbPath">
            <property name="editable">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="btSelPath">
            <property name="text">
             <string>...</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <resources />
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>DlgAddImageClass</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>342</x>
     <y>37</y>
    </hint>
    <hint type="destinationlabel">
     <x>293</x>
     <y>6</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>DlgAddImageClass</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>362</x>
     <y>71</y>
    </hint>
    <hint type="destinationlabel">
     <x>187</x>
     <y>9</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
